panel/src/components/Layout/Panel.vue

```diff
@@ -7,7 +7,7 @@
 		:data-role="role"
 		:data-translation="$translation.code"
 		:data-user="user"
-		:dir="$translation.direction"
+		:dir="dir"
```

Comment:
Do we even need to set the attribute here if we already set it on `<body>`?

Reply:
I didn't want to introduce a breaking change here. Those attributes are promoted as ways to handle custom styles in the panel.css. It's unlikely, but if someone does `.k-panel[dir=rtl] { }` this would break now.